content/knowledge/flare_architecure.md

```diff
@@ -0,0 +1,195 @@
+Validators on the Flare Network operate on the ecosystem as well as the integrated blockchains’ ecosystems.
+
+### Voting
+
+There are three phases of the state connector voting protocol:
+
+* Request
+* Commit
+* Reveal
+
+#### Request Phase
+
+At any point in time, any user can submit a request to the state connector contract to have an event proven.
+The window in time that this request enters the network state is known as the request phase from its perspective.
+
+#### Commit Phase
+
+During the next window of time, attestation providers have the opportunity to commit a hidden vote regarding their belief in the outcome of the events requested in the previous phase.
+Anyone may operate as an attestation provider without any capital requirement, but a default incentivised set is used as the minimal requirement for passing a vote about the events in the previous set.
+
+#### Reveal Phase
+
+Finally, in the next window of time, attestation providers reveal their votes that they committed to in the previous round.
+Once this reveal phase concludes and the next phase begins, the revealed votes are automatically counted and all valid events become immediately available to all contracts on Flare.
+Once these criteria are met and that data is available, the stored transactions in the storage contract are now available to be referenced by any other contract on the Flare Network.
+
+### Branching
+
+The state connector branching protocol protects Flare against incorrect interpretation of real-world events, proactively, such that there are never any rollbacks on the Flare blockchain state.
+Instead of having rollbacks, contention on state correctness is handled via automatic state branching into a correct and incorrect path.
+The security assumption is that if you as an independent node operator are following along with the correct real-world state, then you will always end up on the correct branch of the blockchain state.
+
+## Flare Time Series Oracle (FTSO)
+
+The [FTSO](https://flare.xyz/ftso-a-breakdown/) provides externally sourced data estimates (e.g. the current price of XRP) to the Flare Network in a decentralized manner.
+It does so by leveraging the distributed nature of the network and its participants.
+At the moment, only Spark holders get to vote; in the future, each asset's voting power will be split 50-50 between Spark holders and the holders of the particular asset.
+
+Signal providers exist in the off-chain to collect data, such as asset prices, and deliver it to the FTSO.
+As an example, a provider may collect the price of XRP/$ from three different exchanges, average the price and have this price estimate ready for the FTSO every voting round to submit.
+Signal providers on the Flare Network are responsible for delivering continuous and accurate data estimates to the FTSO to represent off-chain values on the Flare Network.
```

Comment:
IMO this should be the first sentence for signal providers, because it explains what are they